src/primitives/block.h

```diff
@@ -65,18 +65,9 @@ class CBlockHeader
 
     uint256 GetHash() const
     {
-        return hash_Argon2d(BEGIN(nVersion), END(nNonce), 1);
+        return hash_Argon2d(UVOIDBEGIN(nVersion), 1);
```

Comment:
Why did we get rid of the variable length data hash?  This seems like a step back.

Reply:
That was part of rolling back Wolf0's work. However, that part individually can actually stay as BEGIN and END and not be reverted.